### DISCUSSION TITLE
webstorm: program does not start (seccomp)

### DISCUSSION THREAD

**Discussion by csillag** [netblue30/firejail#5804] [Q&A] [answered]
Hi, I have been using JetBrains WebStorm with firejail (0.9.72) for a while, with the following config.

```

# Parts redacted to avoid leaking sensitive data

caps.keep sys_chroot
machine-id
no3d
nodvd
nogroups
nonewprivs
noroot
nosound
notv
nou2f
novideo
protocol unix,inet,inet6
seccomp
private-dev
dbus-user none
dbus-system none
```

After the latest WebStorm upgrade, it won't start.

The line on the console that seems to be relevant is this one:

```
Check failed: sys_chroot("/proc/self/fdinfo/") == 0
```

However I already have `caps.keep sys_chroot`  in my config.
How can I figure out what's wrong?

Thank you for your help.

**Comment by rusty-snake** [comment, marked as answer]
Allow the `chroot` syscall.

```diff
-seccomp
+seccomp !chroot
```

**Comment by csillag** [comment]
Wow, that solved it! Thanks a lot!
